scripts/upgrade

```diff
@@ -183,27 +117,27 @@ ynh_script_progression --message="Installing app..." --weight=1
 
 ynh_use_nodejs
 (
-	cd "$final_path"
+	cd "$install_dir"
 
 	# In case of nodejs upgrade, remove the current node_modules to make sure there are no errors
 	# linked to modules compiled for the previous version.
-	ynh_secure_remove --file="$final_path/node_modules"
+	ynh_secure_remove --file="$install_dir/node_modules"
 
 	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH yarn install
 	ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH yarn cache clean --all
-	ynh_secure_remove --file="$final_path/.cache"
-	ynh_secure_remove --file="$final_path/.yarn"
+	ynh_secure_remove --file="$install_dir/.cache"
+	ynh_secure_remove --file="$install_dir/.yarn"
 )
 
 #=================================================
 # UPDATE A CONFIG FILE
 #=================================================
 ynh_script_progression --message="Updating a configuration file..." --weight=1
 
-ynh_add_config --template="../conf/config.ini" --destination="$final_path/config.ini"
+ynh_add_config --template="../conf/config.ini" --destination="$install_dir/config.ini"
```

Comment:
Same here for config file and salt?

Reply:
Oops I missed those questions, sorry!

> Why are we regenerating the config.ini file? (the config.ini file is preseved during the source upgrade L.50-62)

Good question… Probably just a copy/paste from the installation script…

> Why not use $data_dir path (/home/yunohost.app/kresus) as the data storage?

No particular reason, I guess just a habit from other platforms…